docs/content/en/docs/install/_index.md

```diff
@@ -17,13 +13,43 @@ The steps are:
 
 1. Understand the [Software versions and resources](reqs.md)
    that are required.
+1. Be sure that your cluster includes the components discussed in
+   [Prepare your cluster for KLT](../../install/k8s.md/#prepare-your-cluster-for-klt).
 1. [Bring or create your Kubernetes cluster](k8s.md).
 1. [Replace the default cert-manager](cert-manager.md) (optional).
    This step is only required if you want to replace
    the default KLT cert-manager with another cert-manager.
 1. [Install the Keptn Lifecycle Toolkit](install.md).
+1. [Enable KLT for your cluster](install.md/#enable-klt-for-your-cluster)
```

Comment:
this is the same of the next point, I would suggest to remove it and use the extended name
```suggestion

```